onset_maproom/maproom.py

```diff
@@ -22,6 +22,7 @@
 from shapely import wkb
 from shapely.geometry.multipolygon import MultiPolygon
 from psycopg2 import sql
+import geopandas as gpd
```

Comment:
this has been added but not environment files haven't been changed accordingly.
Which way should be go about this? Add manually to the environment, test that things work, then update environment in version-control? or synchronously?

Reply:
What do you mean by synchronousy?

In this case, geopandas isn't actually being used, so the solution is just to remove this line.

Reply:
Yes in this case I do just need to delete the line since I did not end up using it. However this is a broader discussion I would like to have at some point since it is still confusing to me exactly how we go about updating environments properly. But this conversation seems best to have elsewhere.